modules/doc/content/newsletter/2023/2023_06.md

```diff
@@ -21,6 +21,19 @@ per-subdomain basis: for each subdomain using `RZ`, the axis origin and
 direction are specified. See [Mesh/index.md#coordinate_systems] for more
 information.
 
+### Added HSCoupler2D3D component
+
+The [HSCoupler2D3D.md] component was added to the
+[Thermal Hydraulics module](modules/thermal_hydraulics/index.md). This component
+fully couples a 2D heat structure boundary to a 3D heat structure boundary
+via a simplified gap heat transfer formulation. Using this component can avoid
+the need to create [MultiApps](MultiApps/index.md) for simulations solving
+transient heat conduction between a 3D domain like an assembly matrix and
+2D heat structure components. The main caveat is that the application needs to
+be configured with a sufficient [AD](automatic_differentiation/index.md)
+container size, due to coupling 2D elements to arbitrary numbers of 3D elements.
+See the component documentation for more information.
```

Comment:
Just took a look at the news, since I was pinged about it in my email. Looks good, and thanks for adding content!